Autopilot/Src/gps.cpp

```diff
@@ -0,0 +1,386 @@
+/**
+ * Gps sensor
+ * Author(s): Dhruv Rawat
+ */
+
+#include "gps.hpp"
+#include "UART.hpp"
+#include "../Libraries/STM32F7xx_HAL_Driver/Src/stm32f7xx_hal_uart.c"
```

Comment:
Why are we including .c files ?? That doesn't seem right. Also, this file is very deep inside the hal layer, I'd imagine you'd never have to touch it. Should uart.hpp not be enough ?